### PR TITLE
Aerogear 8137 wl

### PR DESCRIPTION
**Pull request by wei-lee**
## Motivation

https://issues.jboss.org/browse/AEROGEAR-8163

## Verification

1. Provision push, and create an app in the MDC.
2. Create an ios binding, and then an android binding.
3. Try delete the bindings. You should see there are 2 items in the delete drop down of the bound push service. You should be able to delete all of the bindings.

### CODE REVIEW

File: ui/src/components/mobileservices/BindingPanel.test.js [hunk 1]
```diff
@@ -13,6 +14,8 @@ describe('BindingPanel', () => {
   };
   it('should render the Wizard for binding the service', () => {
     const wrapper = shallow(<BindingPanel service={service} />);
+    console.log(wrapper.debug());
```
Comment (aliok) [inline ui/src/components/mobileservices/BindingPanel.test.js]: checked in by mistake?

File: ui/src/components/mobileservices/BindingPanel.test.js [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import { shallow } from 'enzyme';
 import React from 'react';
 import { BindingPanel } from './BindingPanel';
+import { wrap } from 'module';
```
Comment (wojta) [inline ui/src/components/mobileservices/BindingPanel.test.js]: What does this import do?

**Comment by wei-lee** [comment]
@aliok I have implemented all the changes that are required to delete multiple bindings for push. Do you mind reviewing the PR? If you are happy with it, you can merge this PR to your branch and then merge yours. Otherwise I can merge it to master tomorrow morning.

**Comment by aliok** [comment]
Closed by mistake. Can't reopen...

**Comment by aliok** [comment]
@wei-lee verified, rebased and merged https://github.com/aerogear/mobile-developer-console/pull/149

**Pull request closed without merging**